routes/api.php

```diff
@@ -20,7 +23,7 @@
 });
 
 // v2 routes
-$router->group(['prefix' => 'v2', 'middleware' => ['auth.api']], function () {
+$router->group(['prefix' => 'api/v2', 'middleware' => ['auth.api']], function () {
```

Comment:
How is this working without this?

Reply:
Just read your above comment on why it is being added, but I'm confused about how the `api` part of the path was applied before?

Reply:
It was previously applied [in the RouteServiceProvider](https://github.com/DoSomething/rogue/blob/73c28d575e9647254e59e32a9d0a5f0273822166/app/Providers/RouteServiceProvider.php#L66), but since we need some stateless routes _without_ that prefix it seemed to make sense to pull it out. (Plus it was kinda hard to find!)

Reply:
Ahh I see! Should this line still be in there then? `->group(base_path('routes/api.php'));` in `mapApiRoutes()`?

Reply:
Yes, that's what loads this file into the app!

Reply:
OH I see the prefix part now jeez!! That WAS hard to find!